docs/databases/cassandra/how-to-install-apache-cassandra/ubuntu/index.md

```diff
@@ -0,0 +1,299 @@
+keywords: ["cassandra", " apache cassandra", " centos 7", " ubuntu 18.04", " database", " nosql"]
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+published: 2018-09-24
+modified: 2020-01-30
+modified_by:
+  name: Linode
+title: "How to Install Apache Cassandra on Ubuntu 18.04"
+h1_title: "Install Apache Cassandra on Ubuntu 18.04"
+contributor:
+  name: Linode
+external_resources:
+   - '[Cassandra Documentation](http://cassandra.apache.org/doc/latest/)'
+   - '[Cassandra cqlshrc File Configuration Overview](http://docs.datastax.com/en/cql/3.3/cql/cql_reference/cqlshUsingCqlshrc.html)'
+   - '[Cassandra .yaml Configuration File Overview](http://cassandra.apache.org/doc/latest/configuration/cassandra_config_file.html)'
+   - '[Recommended Production Settings For Apache Cassandra](http://docs.datastax.com/en/landing_page/doc/landing_page/recommendedSettings.html)'
+   - '[The Cassandra Query Language (CQL)](http://cassandra.apache.org/doc/latest/cql/index.html)'
+---
+
+After completing this guide, you will have a single-node, production-ready installation of [Apache Cassandra](http://cassandra.apache.org/) hosted on your Linode running Ubuntu 18.04. This tutorial will cover basic configuration options, as well as harden database security.
+
+{{< note >}}
+ In order to successfully execute the commands in this guide, you will need to run them as the `root` user, or log in using an account with root privileges, prefixing each command with `sudo`.
+ {{</ note >}}
+
+## Before You Begin
+
+1. Complete the [Getting Started](/docs/getting-started) guide for setting up a new Linode.
+1. While it is recommended you complete the entire [Securing Your Server](/docs/security/securing-your-server) guide, at  minimum, you should [add a limited user account](/docs/security/securing-your-server/#add-a-limited-user-account).
+
+### Add Repositories and GPG Keys
+
+1.  Add the Java repository:
+
+        sudo add-apt-repository ppa:openjdk-r/ppa
+
+1.  Add the apache repository:
+
+        echo "deb http://www.apache.org/dist/cassandra/debian 40x main" |  sudo tee /etc/apt/sources.list.d/cassandra.list
+
+    {{< note >}}
+You may want to follow the link to the Apache repository to confirm that "40x" is the latest available version.
+    {{< /note >}}
+
+1.  Download the two public keys needed to access these repositories:
+
+        gpg --keyserver pgp.mit.edu --recv-keys 749D6EEC0353B12C
+        gpg --export --armor 749D6EEC0353B12C | sudo apt-key add -
+
+        gpg --keyserver pgp.mit.edu --recv-keys A278B781FE4B2BDA
+        gpg --export --armor A278B781FE4B2BDA | sudo apt-key add -
+
+## Install Cassandra and Supporting Applications
+
+In this section, you will update your Linux system software and install Java along with Cassandra.
+
+1.  Install Cassandra, Java, and NTP. Answer "yes" to the Java installation prompts regarding the license terms (Ubuntu only). NTP will help keep the Cassandra node synced to the correct time.
```

Comment:
Does the "Ubuntu only" part mean there are terms for other distros in the license?

Reply:
This seems to be a standard prompt. Making an edit to reflect this